src/libpriv/rpmostree-postprocess.c

```diff
@@ -1485,7 +1485,11 @@ rpmostree_treefile_postprocessing (int            rootfs_fd,
                                                                 error))
       goto out;
 
-    if (base_version != NULL)
+    if (base_version != NULL && next_version == NULL)
```

Comment:
I don't think we need the `if base_version` part here in the conditional.  I'd do something like:

```
if (base_version == NULL) { g_print ("Ignoring mutate-os-release, no base version"); }
else if (next_version == NULL) { g_print ("ignoring, no next version"); }
else {
 ...
}
```
?